### PR TITLE
skip TLS verification during conformance tests

### PR DESCRIPTION
**Pull request by imjasonh**
It can be useful in tests like this to point the test at an HTTPS endpoint, even though it won't actually serve valid HTTPS, for example because it's running in a test environment.

Making the test skip TLS verification should make it easier to run in such test environments. Conformance doesn't rely on TLS verification itself (explicitly, anyway), and any TLS configuration should be tested outside of OCI conformance, so I don't consider this a regression of the strictness of the test.

### CODE REVIEW

**Review by jdolitsky — APPROVED** [review]

**Review by jdolitsky — DISMISSED** [review]

File: conformance/go.mod [hunk 1]
```diff
@@ -3,21 +3,23 @@ module github.com/opencontainers/distribution-spec/conformance
 go 1.17
 
 require (
-	github.com/bloodorangeio/reggie v0.5.0
+	github.com/bloodorangeio/reggie v0.6.0
 	github.com/google/uuid v1.3.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.16.0
+	github.com/onsi/gomega v1.27.4
```
Comment (sudo-bmitch) [inline conformance/go.mod:9]: We need to update the Dockerfile for this change. We currently use Go 1.17, and this change requires at least 1.18. The following worked for my tests:

```
$ git diff
diff --git a/conformance/Dockerfile b/conformance/Dockerfile
index c462516..249c203 100644
--- a/conformance/Dockerfile
+++ b/conformance/Dockerfile
@@ -1,7 +1,7 @@
 # ---
 # Stage 1: Install certs and build conformance binary
 # ---
-FROM docker.io/golang:1.17.1-alpine3.14 AS builder
+FROM docker.io/golang:1.19.7-alpine3.17 AS builder
 ARG VERSION=unknown
 ARG GO_PKG=github.com/opencontainers/distribution-spec/conformance
 RUN apk --update add git make ca-certificates && mkdir -p /go/src/${GO_PKG}
```

**Review by sudo-bmitch — CHANGES_REQUESTED** [review]

**Review by mikebrow — APPROVED** [review]
LGTM

File: conformance/go.mod (hunk 1 above)
Comment (jdolitsky) [reply conformance/go.mod:9]: missing dockerfile change?

Comment (sudo-bmitch) [reply conformance/go.mod:9]: @imjasonh I'm not seeing the change, give it another git push?

**Review by jdolitsky — DISMISSED** [review]

**Review by jdolitsky — APPROVED** [review]

**Review by sudo-bmitch — APPROVED** [review]

**Pull request merged**